packages/editor/templates/Coin/README.md

```diff
@@ -52,24 +52,25 @@ The `mint` function sends an amount of newly created coins to another address. T
 
 Anyone can use the `send` (who already has some of these coins) to send coins to anyone else. If the sender does not have enough coins to send, the `require` call fails and provides the sender with an appropriate error message string.
 
-### Configure
+### 1. Configure
 
 ![Configure the contract](./images/hw-configure.png)
-Clicking the disclosure triangle next to the contract name reveals three sub-sections. In the _Configure_ sub-section you can change the name of the contract and set the initial values sent to the constructor arguments. In this example, the string displayed in the front end interface.
 
-### Compile
+Configuring the contract allows you to set the name of the contract and the initial values sent to the constructor as arguments. You can configure the contract by clicking on the gear icon in the left panel.
```

Comment:
```suggestion
Configuring the contract allows you to set the name of the contract and the initial values sent to the constructor as arguments. You can configure the contract by clicking on the gear icon in the left panel. In this example, it configures the string displayed in the front end interface.
```

Think it would still be nice to specify  what people are configuring, especially as it's the beginner tutorial.

Reply:
Agree but in this Coin example, there's nothing to actually configure (the constructor doesn't accept any arguments. I think you're referring to the HelloWorld example here. I'll add this sentence there & a separate explanation here.